project.css

```diff
@@ -275,10 +370,44 @@ a {
   padding: 12px 80px;
 }
 
+.return-button:hover {
+  background-color: var(--purple-hover);
+}
+
 .continue-button {
   padding: 12px 65px;
 }
 
+.continue-button:hover {
+  background-color: var(--black-hover);
+}
+
+.category-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: white;
```

Comment:
**Suggestion(non-blocking)**: While I think it's fine here, it is a good idea to consistently use one way of adding colors (rgba/hex etc). Also, maybe adding these as variables that can be used everywhere is also a good idea (primary-background, primary-color etc)